tests/component/sections/shared/social-share-modal/SocialShareModal.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import { SocialShareModal } from '@/sections/shared/social-share-modal/SocialShareModal'
+
+describe('SocialShareModal', () => {
+  it('should render the component title, help text and 3 social links', () => {
+    cy.mount(
+      <SocialShareModal
+        show={true}
+        title="The Title"
+        helpText="The Help Text"
+        shareUrl="shareUrl"
+        handleClose={() => {}}
+      />
+    )
+
+    cy.findByText('The Title').should('exist')
+    cy.findByText('The Help Text').should('exist')
+    cy.get('a').should('have.length', 3)
```

Comment:
we could also check if each social link has correct href, and check when` show = {false}`

Reply:
Thanks, will add links assertion, the close and open functionality (`show` boolean) is tested in the collection page spec